wallix/README.adoc

```diff
@@ -8,7 +8,7 @@ See repository main README for information about licensing, contribution, and ot
 // ====doc====
 [Wallix Bastion-plugin]
 
-= Bastion Wallix integration plugin with Rudder
+= Wallix Bastion integration plugin with Rudder
```

Comment:
```suggestion
= Wallix Bastion
```

Reply:
![menu](https://user-images.githubusercontent.com/329388/85157291-cde6cb00-b24a-11ea-8cbd-a4f09296699e.png)
It's what will appear in the menu